scripts/zones/Eastern_Adoulin/npcs/Ujlei_Zelekko.lua

```diff
@@ -0,0 +1,71 @@
+-----------------------------------
+-- Area: Eastern Adoulin (257)
+--  NPC: Ujlei Zelekk
+-- Type: Peacekeepers Coalition Guard
+-- !pos -102.754 -0.65 16.161 257
+-- !gotoid 17830177
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/npc_util')
+require('scripts/globals/settings')
+require('scripts/globals/extravaganza')
+-----------------------------------
+local entity = {}
+
+local items =
+{
+    [   4] = { cost = 2500, id = xi.items.COALITION_POTION },
+    [ 260] = { cost = 2500, id = xi.items.COALITION_ETHER },
+    [ 516] = { cost =   10, id = xi.items.SCROLL_OF_INSTANT_WARP },
+    [ 772] = { cost =   10, id = xi.items.SCROLL_OF_INSTANT_RERAISE },
+    [1028] = { cost = 1000, id = xi.items.SCROLL_OF_INSTANT_PROTECT },
+    [1284] = { cost = 1000, id = xi.items.SCROLL_OF_INSTANT_SHELL },
+    [1540] = { cost = 1000, id = xi.items.SCROLL_OF_INSTANT_STONESKIN },
+    [1796] = { cost = 2000, id = xi.items.CIPHER_OF_MARGRETS_ALTER_EGO },
+    [2052] = { cost = 2000, id = xi.items.CIPHER_OF_AMCHUCHUS_ALTER_EGO },
+    [2308] = { cost = 2000, id = xi.items.CIPHER_OF_MORIMARS_ALTER_EGO },
+}
+
+local active = xi.extravaganza.campaignActive()
```

Comment:
`active` isn't very descriptive in the lua's "global" space, You could move this to the `onTrigger` function and it would be fine unchanged because it is then used nearly immediately. This follows the pattern of the rest of your PR as well.